### PR TITLE
Feed

### PR DESCRIPTION
**Pull request by A-n-u-O**
image
Screenshot (63)

### CODE REVIEW

**Review by Boye-dev — COMMENTED** [review]
Update comment UI 
Remove Modal

**Comment by A-n-u-O** [comment]
image
Screenshot (66)

File: src/Components/Feed.tsx [hunk 1]
```diff
@@ -82,33 +90,105 @@ const Feed = () => {
...
+  };
+  const addCommentNumber = (index: number) => {
+    let commentNo = index + 1;
+    return commentNo;
+  };
+  const CommentNumber = ({ commentNo }: { commentNo: number }) => (
+    <Box component="span">{commentNo} comments</Box>
+  );
+
+  const Comment = ({
```
Comment (Boye-dev) [inline src/Components/Feed.tsx:123]: Move these components to their own seperate files. You can add it to your components folder

Comment (A-n-u-O) [reply src/Components/Feed.tsx:123]: okay

**Pull request merged**